pkg/model/fmt.go

```diff
@@ -0,0 +1,24 @@
+package model
+
+import (
+	"regexp"
+	"strings"
+)
+
+var goStructTypeParameterRegex = regexp.MustCompile(`\[go\.shape\..*\]`)
+
+func DropGoTypeParameters(input string) string {
+	matchesIndices := goStructTypeParameterRegex.FindAllStringIndex(input, -1)
+	if len(matchesIndices) == 0 {
+		return input
+	}
+	var modified strings.Builder
+	prevEnd := 0
+	for _, indices := range matchesIndices {
+		start, end := indices[0], indices[1]
+		modified.WriteString(input[prevEnd:start] + "[...]")
+		prevEnd = end
+	}
+	modified.WriteString(input[prevEnd:])
+	return modified.String()
+}
```

Comment:
Note that we don't know if it is a Go name therefore it is applied to all nodes. `ReplaceAllString` would look better here but it allocates a new string.